tests/hypothesis_strategies/utils.py

```diff
@@ -0,0 +1,36 @@
+"""Hypothesis strategies for generating utility objects."""
+
+import hypothesis.strategies as st
+from hypothesis import assume
+
+from baybe.utils.interval import Interval
+
+
+@st.composite
+def interval(
+    draw: st.DrawFn,
+    *,
+    exclude_bounded: bool = False,
+    exclude_half_bounded: bool = False,
+    exclude_fully_unbounded: bool = False,
+):
+    """Generate :class:`baybe.utils.interval.Interval`."""
+    assert not all(
+        (exclude_bounded, exclude_half_bounded, exclude_fully_unbounded)
+    ), "At least one Interval type must be allowed."
+
+    # Create interval from ordered pair of floats
+    bounds = (
+        st.tuples(st.floats(), st.floats()).map(sorted).filter(lambda x: x[0] < x[1])
+    )
+    interval = Interval.create(draw(bounds))
+
+    # Filter excluded intervals
+    if exclude_bounded:
+        assume(not interval.is_bounded)
+    if exclude_half_bounded:
+        assume(not interval.is_half_bounded)
+    if exclude_fully_unbounded:
+        assume(interval.is_left_bounded or interval.is_right_bounded)
```

Comment:
Hey @AVHopp 👋🏼 Now that we follow the mathematical nomenclature, I am struggling a bit with finding a good name for the situation when an interval is neither left nor right bounded, i.e. if it's simply the whole number line. What would be a good name for such a `property`? In fact, I'd like to add such a property, because I already made the mistake of combining the existing properties in the wrong way to recreate it... A simple `is_unbounded` would be wrong because that could mean both half-bounded and not bounded on both ends. Suggestions?

Reply:
The only thing I could come up with is `is_fully_bounded`. Perhaps @Scienfitz also has a suggestion?

Reply:
I think your suggestion is good and gets across the point, looking online there seems to be no proper definition that is equivalent to `fully unbounded`

Reply:
Alright, thanks! Have added the `is_fully_unbounded` property, which is now used in the corresponding hypothesis strategy. BTW: also fixed the deduplication error in the parameters strategy that made our regular pipeline check fail.

Reply:
No. *GitHub* happens :D